src/main/Controller/SvgFileCreation.java

```diff
@@ -0,0 +1,8 @@
+package Controller;
+
+public interface SvgFileCreation {
+	public void createEmptyFile();
```

Comment:
I would say createFile rather thant Empty File. Here, their is no need to create an empty file